main/signers/lattice/Lattice/index.ts

```diff
@@ -268,21 +265,25 @@ export default class Lattice extends Signer {
 
   async signTransaction (index: number, rawTx: TransactionData, cb: Callback<string>) {
     try {
+      const connection = this.connection as Client
       const compatibility = signerCompatibility(rawTx, this.summary())
       const latticeTx = compatibility.compatible ? { ...rawTx } : londonToLegacy(rawTx)
 
-      const signedTx = await sign(latticeTx, async tx => {
+      const signedTx: any = await sign(latticeTx, async tx => {
         const unsignedTx = this.createTransaction(index, rawTx.type, latticeTx.chainId, tx)
         const signOpts = { currency: 'ETH', data: unsignedTx }
```

Comment:
Please use `getFwVersion` to inform how you will build this request.

Just because we worked to sanitize outputs from 0.15.0 bridged ETH requests doesn't mean we shouldn't put in the legwork here to avoid making bridged requests when fw is >=0.15.0. We don't want to maintain that shim forever.